package.json

```
@@ -2,6 +2,7 @@
   "name": "galactics-bot-enhanced",
   "version": "1.0.0",
   "description": "Galactics Official bot",
+  "type": "module",
```

```suggestion
  "type": "module",
```
This shouldn't be added